arc/vocabulary.csv

```diff
@@ -0,0 +1,2 @@
+term,type,label,description,domain,range
+columnIndex,Property,columnIndex,"""Index of a column in a tabular information representation.""",PropertyValue,Integer
```

Comment:
The description could be related a bit better to the class it's applied to, e.g.

```suggestion
columnIndex,Property,columnIndex,"""Index of a column the property occupies in a tabular information representation.""",PropertyValue,Integer
```

(inspired by the other attributes of https://schema.org/PropertyValue)